foundation_cms/templates/patterns/components/donate_banner.html

```diff
@@ -0,0 +1,21 @@
+{% load settings_value i18n static wagtailimages_tags %}
+
+<div data-banner-id="{{ banner.id }}"
+     data-banner-title="{{ banner.title }}"
+     data-banner-name="{{ banner.name }}"
+     data-banner-variant-version="{{ banner.variant_version }}"
+     data-banner-active-ab-test="{{ banner.active_ab_test }}"
+     class="donate-banner"
+     style="border: 1px solid black; padding-left:20px;"
+>
+    <p><b>Banner Name:</b> {{ banner.name }}</p>
+    <p><b>Banner Title:</b> {{ banner.title }}</p>
+    <p><b>Banner Subtitle:</b> {{ banner.subtitle }}</p>
+    <p><b>Banner Foreground Image:</b> {{ banner.foreground_image }}</p>
+    <p><b>Banner Background Image:</b> {{ banner.background_image }}</p>
+    <p><b>Banner Text Color:</b> {{ banner.text_color }}</p>
+    <p><b>Banner Button Color:</b> {{ banner.button_color }}</p>
+    <p><b>Banner Button Text and Link:</b> <a id="banner-cta-button" href="{{ banner.cta_link }}">{{ banner.cta_button_text }}</a></p>
+    <p><b>Banner Active AB Test (If Applicable):</b> {{ banner.active_ab_test }}</p>
+    <p><b>Banner Variant Version (If active AB test):</b> {{ banner.variant_version }}</p>
+</div>
```

Comment:
As mentioned in the PR description, this template is very basic and just showing the donate banner data. We will update this in a later PR once we get designs/field requests from stakeholders.